### PR TITLE
Qradar generic mapping

### PR DESCRIPTION
**Pull request by David-BMS**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/28939

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by David-BMS** [comment]
@yaakovi the review is just for the scripts and creating Unit tests etc to make it content worthy

File: Packs/QRadar/Scripts/QRadarFetchedEventsSum/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+This display the amount of fetched events vs the total amount of events in the offense.
```
Comment (David-BMS) [inline Packs/QRadar/Scripts/QRadarFetchedEventsSum/README.md:1]: @yaron-libman

File: Packs/QRadar/Scripts/QRadarMagnitude/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+This enables to color the field according to the magnitude. The scale is
```
Comment (David-BMS) [inline Packs/QRadar/Scripts/QRadarMagnitude/README.md:1]: @yaron-libman

File: Packs/QRadar/Scripts/QRadarPrintAssets/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+This script prints the assets fetched from the offense in a table format.
```
Comment (David-BMS) [inline Packs/QRadar/Scripts/QRadarPrintAssets/README.md:1]: @yaron-libman

File: Packs/QRadar/Scripts/QRadarPrintEvents/README.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+This script prints the events fetched from the offense in a table format.
```
Comment (David-BMS) [inline Packs/QRadar/Scripts/QRadarPrintEvents/README.md:1]: @yaron-libman

**Comment by yaron-libman** [comment]
Review complete

**Review by yaakovi — COMMENTED** [review]
@michalgold scripts are done.
@David-BMS let me know if anything starts misbehaving

**Comment by David-BMS** [comment]
@kirbles19 
Reminding this is for 6.0.0 and QRadar V2 integration

File: Packs/Asset/pack_metadata.json [hunk 5]
```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Asset",
+    "description": "Base pack for any packs using Assets.",
```
Comment (David-BMS) [inline Packs/Asset/pack_metadata.json:3]: @yaron-libman @kirbles19 
Hi we decided to create a new pack with incident fields just for assets
It contains only the previous incident fields from before
Please review if needed

Comment (michalgold) [inline Packs/Asset/pack_metadata.json:3]: using Asset fields.

Comment (David-BMS) [reply Packs/Asset/pack_metadata.json]: Done

**Review by michalgold — APPROVED** [review]

**Pull request merged**